modules/chapter1/pages/section3.adoc

```diff
@@ -1,4 +1,2 @@
 = Section 3
 
-This is _Section 3_ of _Chapter 1_ in the *hello* quick course....
```

Comment:
What's the point of this section? Can we remove it?

Reply:
Removed